src/components/Card.tsx

```
@@ -59,169 +34,143 @@ export const Card: React.FC<CardProps> = ({
-      data-card-id={id}
-      data-stage={stage}
-      onClick={onClick}
+      className={`card ${card.isBlocked ? 'card-blocked' : ''} ${isCompleted ? 'card-completed' : ''} ${isDragOver ? 'card-drag-over' : ''}`}
+      data-testid="card"
+      data-card-id={card.id}
+      data-stage={card.stage}
+      onClick={onCardClick ? () => onCardClick(card.id) : undefined}
       onDragOver={handleDragOver}
       onDragLeave={handleDragLeave}
       onDrop={handleDrop}
     >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)